pairing/bls12381/suite_test.go

```diff
@@ -0,0 +1,404 @@
+package bls12381_test
+
+import (
+	"bytes"
+	"crypto/cipher"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v3"
+	this "go.dedis.ch/kyber/v3/pairing/bls12381"
+	"go.dedis.ch/kyber/v3/util/random"
+)
+
+// Code extracted from kyber/utils/test
+// TODO: expose API in forked drand/kyber
```

Comment:
What is this about? Do we need this before merging? Or is this a post-merge operation? Which API?